decidim-accountability/app/cells/decidim/accountability/project_cell.rb

```diff
@@ -35,6 +35,14 @@ def scope
 
       def tab_panel_items
         [
+          {
+            enabled: ResultHistoryCell.new(result).send(:history_items).any?,
```

Comment:
```suggestion
            enabled: ResultHistoryCell.new(result).history_items.any?,
```
Let's just make the method public

Reply:
Thanks, It's done.